_board/pimoroni_tinyfx.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: download
+board_id: "pimoroni_tinyfx"
+title: "Tiny FX Download"
+name: "Tiny FX"
+manufacturer: "Pimoroni"
+board_url:
+ - "https://shop.pimoroni.com/products/tinyfx"
+board_image: "pimoroni_tinyfx.jpg"
+date_added: 2021-09-02
+family: raspberrypi
+
+features:
+  - USB-C
+  - STEMMA QT/QWIIC
```

Comment:
Could this have `Speaker` listed? Or is that only for boards with an attached speaker, not just a speaker output?